doc/policy_composer.md

```diff
@@ -22,4 +22,27 @@ Configuration | Mandatory? | Description
 --------------|------------|-------------
 handle_unknown_perms|Yes|This is how permissions missing from the policy will be handled when loading the policy. The options are allow, deny, and reject. [handleunknown](https://github.com/SELinuxProject/selinux/blob/master/secilc/docs/cil_policy_config_statements.md#handleunknown)
 system_type|No|This controls whether MLS/MCS controls are enabled. The only currently supported option is standard. Options for mls and mcs will be added in the future. The default is standard. [MLS/MCS](https://github.com/SELinuxProject/selinux-notebook/blob/main/src/mls_mcs.md)
-monolithic|No|This is the option for choosing to build monolithic or modular policy. The options are true and false, with true being for a monolithic build and false for modular. The default is true. [Monolithic and Modular policies](https://github.com/SELinuxProject/selinux-notebook/blob/main/src/types_of_policy.md#monolithic-policy)
\ No newline at end of file
+monolithic|No|This is the option for choosing to build monolithic or modular policy. The options are true and false, with true being for a monolithic build and false for modular. The default is true. [Monolithic and Modular policies](https://github.com/SELinuxProject/selinux-notebook/blob/main/src/types_of_policy.md#monolithic-policy)
+
+# System Building
+There are 3 options for system building.
+
+(1) compile_combined  
+This is the default when casc is ran without the -s flag. It compiles all
```

Comment:
All systems, or all defined policy?  In other words, if I have a resource foo and it is included in no modules or systems, it would still be included with compile_combined, right?